README.md

```diff
@@ -122,6 +125,153 @@ url->set_hash("is-this-the-real-life");
 // url->get_hash() will return "#is-this-the-real-life"
 ```
 
+### Command line options
```

Comment:
I think we can move command line options to `docs/cli.md` file, and add a link to the file from the README.